### PR TITLE
Fix test issues

### PR DESCRIPTION
**Pull request by vandenman**
Fixes https://github.com/jasp-stats/jasp-test-release/issues/2234
Fixes https://github.com/jasp-stats/jasp-test-release/issues/2236
Closes https://github.com/jasp-stats/jasp-test-release/issues/2144

@Kucharssim I changed 'degree' to 'strength' as that is the displayed name. I changed this in the upgrader, but do I also need to update anything in https://github.com/jasp-stats/jaspNetwork/tree/master/tests/upgrades/0.16.4 for this?

### CODE REVIEW

File: R/networkanalysis.R [hunk 1]
```diff
@@ -475,12 +475,12 @@ NetworkAnalysis <- function(jaspResults, dataset, options) {
...
     return()
 
-  measuresToShow <- unlist(options[c("betweenness", "closeness", "degree", "expectedInfluence")], use.names = FALSE)
+  measuresToShow <- unlist(options[c("betweenness", "closeness", "strength", "expectedInfluence")], use.names = FALSE)
   hasMeasures <- any(measuresToShow)
 
   width <- if (hasMeasures) 120 * sum(measuresToShow) else 120
   plot <- createJaspPlot(title = gettext("Centrality Plot"), position = 52, width = width,
-                         dependencies = c("centralityPlot", "betweenness", "closeness", "degree", "expectedInfluence"))
+                         dependencies = c("centralityPlot", "betweenness", "closeness", "strength", "expectedInfluence"))
```
Comment (vandenman) [inline R/networkanalysis.R:483]: figure width is not good.

**Review by FBartos — COMMENTED** [review]
apart from the 2-3 figures width change bug the previous bugs are fixed

**Comment by vandenman** [comment]
@FBartos the last commit should fix the issue, if the plot is too small then the main plot area disappears because that's what ggplot like to do... :shrug:

**Review by FBartos — APPROVED** [review]

**Comment by FBartos** [comment]
I also found this: https://github.com/jasp-stats/INTERNAL-jasp/issues/2227
but probably not worthy pursuing for release

**Comment by vandenman** [comment]
No that's impossible/ very hard to do. Basically, you have to render the plot twice. Once to figure out the size of the legend, which depends on the factor labels, font, and font size, and then a second time where you increase the width to account for the legend width.

**Pull request merged**